Makefile

```diff
@@ -126,6 +126,17 @@ deps: | deps-common bottles
 
 update: | update-common
 
+QML_DEBUG ?= false
```

Comment:
Can we make this `true` by default when we do a debug build?

Reply:
Unfortunately, nim-status-client is built in debug mode by default and some extra job needs to be done to properly configure it for release :man_shrugging:

> Presently the `RELEASE` variable should not be set on the command-line
nor in CI as more work needs to be done to toggle the proper compiler flags.

> At present, production builds and development
builds are all debug builds, but that will likely change in the future

source: https://github.com/status-im/status-desktop/commit/c03a086c01e4e2bd978a8ae8c54d9f95128c4971